cfgov/jinja2/v1/_layouts/base-common.html

```diff
@@ -188,27 +191,18 @@
       Turn off JavaScript for browsers that don't support ECMAScript 5 features
       (e.g. Internet Explorer 8)
       by reversing no-js/js CSS class change made by modernizr.
-      The following ECMAScript 5 feature checks come from
+      The ECMAScript 5 feature checks are listed in
       https://github.com/Modernizr/Modernizr/tree/master/feature-detects/es5
     #}
     <script>
         var modernizr = window.Modernizr;
```

Comment:
Wrap this like  so the variables don't have global scrope
```
 <script>
      !function(){
        var modernizr = window.Modernizr;
        if ( !( typeof modernizr !== 'undefined' && modernizr.es5 ) ) {
          var docElement = document.documentElement;
          docElement.className = docElement.className.replace( /(^|\s)js(\s|$)/, '$1no-js$2' );
        }
      }()
    </script>
```

Reply:
We'll want modernizr in the global scope if we had these checks like `modernizr.es5` elsewhere, but as we don't, this should be good 👍 
Fixed.